src/components/ArticlesList.jsx

```diff
@@ -2,32 +2,36 @@ import Article from "./Article"
 import { getArticles } from "../utils/api"
```

Comment:
Currently, if a user goes to a nonexistent path, for example `/katherine`, they aren't given any feedback
Can you make it so they get some sort of indication when they are on a non-existent path?